service/history/archival/archiver_test.go

```diff
@@ -53,33 +55,23 @@ func TestArchiver(t *testing.T) {
 		ArchiveVisibilityErr error
 		RateLimiterWaitErr   error
 		Targets              []Target
+		SearchAttributes     *common.SearchAttributes
+		SearchAttributesErr  error
+		NameTypeMap          searchattribute.NameTypeMap
+		NameTypeMapErr       error
 
 		ExpectArchiveHistory    bool
 		ExpectArchiveVisibility bool
-		ExpectedMetrics         []expectedMetric
```

Comment:
I removed these and the log recorder messages because they're too much of a pain to maintain